src/constants/status_codes.json

```diff
@@ -33,10 +37,25 @@
     "416": "Requested Range Not Satisfiable",
     "417": "Expectation Failed",
     "418": "I'm a teapot",
+    "421": "Misdirected Request",
+    "422": "Unprocessable Entity",
+    "423": "Locked",
+    "424": "Failed Dependency",
+    "425": "Too Early",
+    "426": "Update Required",
+    "428": "Precondition Required",
+    "429": "Too Many Requests",
+    "431": "Request Header Fields Too Large",
+    "451": "Unavailable For Legal Reasons",
     "500": "Internal Server Error",
     "501": "Not Implemented",
     "502": "Bad Gateway",
     "503": "Service Unavailable",
     "504": "Gateway Timeout",
-    "505": "HTTP Version Not Supported"
+    "505": "HTTP Version Not Supported",
```

Comment:
These looks good, just wondering which official spec source you used to write all these since I couldn't find all of these in the https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html spec page.

Reply:
I got them from MDN. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status